### PR TITLE
feat: [CVNG-4073]: Replaced GCODashboardWidgetMetricNav

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/health-source/connectors/GCOMetricsHealthSource/GCOMetricsHealthSource.tsx [hunk 1]
```diff
@@ -251,6 +259,12 @@ export function GCOMetricsHealthSource(props: GCOMetricsHealthSourceProps): JSX.
           setIsIdentifierEdited(true)
         }
 
+        const dashboard = data.selectedDashboards.map((item: { path: string; name: string }) => {
+          return { itemId: item.path, title: item.name }
+        })
+
+        const isDashdoardEmpty = !Object.values(dashboard).filter(item => !item).length
```
Comment (praneshg239) [inline src/modules/85-cv/pages/health-source/connectors/GCOMetricsHealthSource/GCOMetricsHealthSource.tsx:266]: A suggestion here:

rather writing like
```js
const isDashdoardEmpty = !Object.values(dashboard).filter(item => !item).length
```
we can write it as
```js
const isDashdoardItemsPresent = Boolean(Object.values(dashboard).filter(item => item).length)
```

This avoids the complications of using `!items` and later using `!` to cast it.

Comment (praneshg239) [reply src/modules/85-cv/pages/health-source/connectors/GCOMetricsHealthSource/GCOMetricsHealthSource.tsx:266]: Also I feel it is much declarative.

**Review by praneshg239 — DISMISSED** [review]

**Comment by deepesh-ui** [comment]
retrigger jest

**Comment by deepesh-ui** [comment]
retrigger jest

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**